lib/katello/engine.rb

```diff
@@ -161,7 +161,7 @@ def find_assets(args = {})
       ::OperatingsystemsController.send :include, Katello::Concerns::OperatingsystemsControllerExtensions
       ::HostsController.send :include, Katello::Concerns::HostsControllerExtensions
       ::Containers::StepsController.send :include, Katello::Concerns::Containers::StepsControllerExtensions
-
+      ::SmartProxiesController.send :include, Katello::Concerns::SmartProxiesControllerExtensions
```

Comment:
I think the blank line that was removed was there to separate controller extensions from the next section of code that deals with facts.